[en] update rdmausage (#4975)
DaoCloud/DaoCloud-docs: docs/en/docs/network/config/rdmatype.md

# RDMA Usage Comparison

> This section introduces the comparison of RDMA network usage with different CNIs, as follows:

Reviewer: ```suggestion

This section introduces the comparison of RDMA network usage with different CNIs, as follows:
```
正文前留一个空行，中文的不良不要蔓延过来

> | Functionality Comparison | MACVLAN/IPVLAN +RoCE Network Passthrough | SRIOV +RoCE Network Passthrough | SRIOV +Infiniband Network Passthrough |

Reviewer: ```suggestion
| Functionality Comparison | Passthrough via MACVLAN/IPVLAN + RoCE | Passthrough via SRIOV + RoCE | Passthrough via SRIOV + InfiniBand |
```
中文也要修改，`+` 前后留1个空格

| ------------------ | ----------------------------- | ---------------------- | -------------------------- |
| Performance | Good | Very Good | Best |

> | Number of Pods Supported per Node | Unlimited | Number of Restricted SmartNIC Slicing VFs | Number of Restricted SmartNIC Slicing VFs |

Reviewer: ```suggestion
| Number of Pods Supported per Node | Unlimited | Number of Split Virtual Functions on a Restricted SmartNIC | Number of Split Virtual Functions on a Restricted SmartNIC |
```
这些词像 Title 一样大写，会不会有点奇怪

> | Switch | Ethernet Switch | Ethernet Switch | IB Switch | 

Reviewer: ```suggestion
| Switch | Ethernet Switch | Ethernet Switch | InfiniBand Switch | 
```
有足够的空间，用全称，中文也要修改

# RDMA Usage Comparison

*[VF]: Virtual Function, a lightweight PCIe function on a network adapter that supports single root I/O virtualization (SR-IOV)

> This section introduces the comparison of RDMA network usage with different CNIs, as follows:

Reviewer: ```suggestion
This section provides a comparative analysis of RDMA network usage across various CNIs (Container Network Interfaces).
```

> 

Reviewer: 有了 markdownlint 后，不要这种连续的空行，空一行就够了
